snapshots/lcow/lcow.go

```diff
@@ -380,7 +400,7 @@ func (s *snapshotter) openOrCreateScratch(ctx context.Context) (_ *os.File, err
 			LogFormat: runhcs.JSON,
 			Owner:     "containerd",
 		}
-		if err := rhcs.CreateScratch(ctx, scratchTempPath); err != nil {
+		if err := rhcs.CreateScratch(ctx, scratchTempPath, sizeGB); err != nil {
```

Comment:
Oh and this will have to be: https://github.com/microsoft/hcsshim/blob/master/pkg/go-runhcs/runhcs_create-scratch.go#L44